packages/core/src/data-manager/DataManager.ts

```
@@ -306,6 +306,19 @@
     }
   }
 
+  static updateEnv(newEnv: ConnectSettings['env']) {
+    if (this.settings) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)